src/main/java/com/faforever/client/achievements/AchievementService.java

```diff
@@ -2,22 +2,113 @@
 
 import com.faforever.client.api.dto.AchievementDefinition;
 import com.faforever.client.api.dto.PlayerAchievement;
+import com.faforever.client.config.CacheNames;
+import com.faforever.client.player.PlayerService;
+import com.faforever.client.remote.AssetService;
+import com.faforever.client.remote.FafService;
+import com.faforever.client.remote.UpdatedAchievementsMessage;
+import com.google.common.annotations.VisibleForTesting;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
```

Comment:
did you click optimize imports before committing?

Reply:
Just did that, and squashed the commits into one.  Problem is all the commits are still listed above (not sure how to remove them).